src/fluree/db/query/exec/eval.cljc

```diff
@@ -117,16 +118,166 @@
   (str/ends-with? s substr))
 
 (defn subStr
-  [s start end]
-  (subs s start end))
+  ([s start]
+   (subStr s start (count s)))
```

Comment:
I think this is confounding length and endpoint. If start is not 1, the second argument to `subStr` will be longer that the length remaining. In any case, the built-in `subs` has an arity that accomplishes this case. i think `(subs s (dec start))` is what you want here.